head_tilt_joint:                        { direction: -1, origin:   0.0 }
head_pan_joint:                         { direction: -1, origin:  45.0 }
left_torso_shoulder_mounting_joint:     { direction:  1, origin: -90.0 }
left_shoulder_mounting_shoulder_joint:  { direction:  1, origin: -90.0 }
left_shoulder_bicep_joint:              { direction:  1, origin:   0.0 }
left_bicep_forearm_joint:               { direction: -1, origin:   0.0 }
left_forearm_wrist_joint:               { direction:  1, origin:   0.0 }
left_wrist_hand_joint:                  { direction:  1, origin:   0.0 }
left_hand_thumb_joint:                  { direction:  1, origin:   0.0 }
right_torso_shoulder_mounting_joint:    { direction: -1, origin:  90.0 }
right_shoulder_mounting_shoulder_joint: { direction: -1, origin:  90.0 }
right_shoulder_bicep_joint:             { direction: -1, origin:   0.0 }
right_bicep_forearm_joint:              { direction:  1, origin:   0.0 }
right_forearm_wrist_joint:              { direction: -1, origin:   0.0 }
right_wrist_hand_joint:                 { direction:  1, origin:   0.0 }
right_hand_thumb_joint:                 { direction:  1, origin:   0.0 }
eyebrows_joint:                         { direction:  1, origin:   0.0 }
mouth_top_joint:                        { direction:  1, origin:   0.0 }
mouth_bottom_joint:                     { direction:  1, origin:   0.0 }
